src/Graph/GraphPrinter.php

```diff
@@ -224,6 +241,12 @@ public function getParamDefinitions( array $definitions ) {
 			'values'  => [ 'LR', 'RL', 'TB', 'BT' ],
 		];
 
+		$params['arrowhead'] = [
+			'default' => 'normal',
+			'message' => 'srf-paramdesc-arrowhead', // TODO: add to i18n en.json de.json qqq.json
+			'values' => self::$ARROW_SHAPES,
+		];
```

Comment:
Probably this should have a default value? Or is it a required parameter?

Reply:
I can't figure out why the tests are failing. When I run them locally, they pass without errors.

The travis-ci says **Undefined index: arrowhead** but how can this be since I've passed the $params-array with the index **arrowhead** .

The default value is already set to "normal" for the arrowhead or do I misunderstand you?

Reply:
Yeah the default value is set now

Reply:
Latest test failure seems unrelated to SRF. I triggered a rebuild on TravisCI